Classes/JBLineChartView.m

```diff
@@ -606,6 +650,30 @@ - (UIColor *)lineChartLinesView:(JBLineChartLinesView *)lineChartLinesView selec
     return kJBLineChartViewDefaultLineSelectionFillColor;
 }
 
+- (CAGradientLayer *)lineChartLinesView:(JBLineChartLinesView *)lineChartLinesView fillGradientForLineAtLineIndex:(NSUInteger)lineIndex
+{
+    if ([self.delegate respondsToSelector:@selector(lineChartView:fillGradientForLineAtLineIndex:)]) {
+        return [self.delegate lineChartView:self fillGradientForLineAtLineIndex:lineIndex];
+    }
+    CAGradientLayer *gradient = [self defaultGradientLayer];
+    gradient.colors = @[(id)kJBLineChartViewDefaultGradientFillStartColor.CGColor, (id)kJBLineChartViewDefaultGradientFillEndColor.CGColor];
+    return gradient;
+}
+
+- (CAGradientLayer *)lineChartLinesView:(JBLineChartLinesView *)lineChartLinesView selectionFillGradientForLineAtLineIndex:(NSUInteger)lineIndex {
+    if ([self.delegate respondsToSelector:@selector(lineChartView:selectionFillGradientForLineAtLineIndex:)]) {
+        return [self.delegate lineChartView:self selectionFillGradientForLineAtLineIndex:lineIndex];
+    }
+    return [self.delegate lineChartView:self fillGradientForLineAtLineIndex:lineIndex];
+}
+
+- (CAGradientLayer *)defaultGradientLayer {
+    CAGradientLayer *gradient = [CAGradientLayer new];
+    gradient.startPoint = CGPointMake(kJBLineChartViewDefaultStartPoint, kJBLineChartViewDefaultStartPoint);
+    gradient.endPoint = CGPointMake(kJBLineChartViewDefaultEndPoint, kJBLineChartViewDefaultEndPoint);
+    return gradient;
```

Comment:
I think you're making a diagonal gradient here. A constant X coordinate would make a vertical gradient. Even if it doesn't matter in the output given the default colors, it may be beneficial to use a pure vertical gradient for when people look at your code to understand how it all works.